.github/workflows/commit-check.yml

```diff
@@ -86,3 +86,42 @@ jobs:
           GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
           CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
           CYPRESS_PROJECT_ID: ${{ secrets.CYPRESS_PROJECT_ID }}
+
+  e2e-test:
+    runs-on: ubuntu-22.04
+    needs: install
+    steps:
+      - name: Checkout local-env
+        uses: actions/checkout@v4
+        with:
+          repository: TourmalineCore/inner-circle-local-env
+
+      - name: Deploy Local Env to Kind k8s
+        uses: devcontainers/ci@v0.3
+        with:
+          runCmd: |
+            kind create cluster --name inner-circle --config kind-local-config.yaml --kubeconfig ./.inner-circle-cluster-kubeconfig
+            export KUBECONFIG=$(pwd)/.inner-circle-cluster-kubeconfig
+            helmfile --environment local --namespace local -f deploy/helmfile.yaml apply
+          push: never
+
+      - name: Checkout ui
+        uses: actions/checkout@v4
+
+      - name: Cypress run
+        uses: cypress-io/github-action@v6
+        with:
+          wait-on: "http://localhost:40100" 
+          spec: "cypress/e2e/compensations-smoke.cy.ts"
+          browser: chrome
+          record: true
+        env:
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+          CYPRESS_RECORD_KEY: ${{ secrets.CYPRESS_RECORD_KEY }}
+          CYPRESS_PROJECT_ID: ${{ secrets.CYPRESS_PROJECT_ID }}
+          CYPRESS_BASE_URL: http://localhost:40100
+          API_ROOT: /api
+          API_ROOT_AUTH: /api/auth
+          LINK_TO_COMPENSATIONS_SERVICE: /compensations
+          USER_LOGIN: ceo@tourmalinecore.com
```

Comment:
Some of these vars, including creds, are already hardcoded in `cypress.config.local-env.ts`, should we duplicate them here?